lib/json2csv.js

```diff
@@ -307,6 +305,8 @@ function createDataRows(originalData, unwindPaths) {
 
       Array.prototype.push.apply(dataRows, dataRow);
```

Comment:
This is actually necessary. Check out [this link](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/push#Merging_two_arrays), but basically we need to merge two arrays (dataRows and dataRow).